src/commands.ts

```diff
@@ -238,20 +256,32 @@ export class CommandCenter {
             if (!repo) {
                 return;
             }
-            const currentRef: string = await this._gitService.getCurrentBranch(repo);
-            window.showQuickPick(selectBranch(this._gitService, repo, true), { placeHolder: `Select a ref to see it's diff with ${currentRef} (${repo.root})` })
-                .then(async item => {
-                    if (item) {
-                        const leftRef = await getRefFromQuickPickItem(item, `Input a ref(sha1) to compare with ${currentRef}`);
-                        if (!leftRef) return;
-                        this._model.filesViewContext = {
-                            repo,
-                            leftRef,
-                            rightRef: currentRef,
-                            specifiedPath: null
-                        };
-                    }
-                });
+            const branchs: QuickPickItem[] = await selectBranch(this._gitService, repo, true)
+            const branchWithCombination: QuickPickItem[] = await branchCombination(this._gitService, repo)
+            const items: QuickPickItem[] = [...branchs, ...branchWithCombination]
+            const placeHolder: string = 'Select a branch to compare...'
+            window.showQuickPick(items, { placeHolder: placeHolder }).then(async item => {
+                if (!item) {
+                    return;
+                }
+                const currentRef: string = await this._gitService.getCurrentBranch(repo);
+                let leftRef = await getRefFromQuickPickItem(item, `Input a ref(sha1) to compare with ${currentRef} or 'ref(sha1)..ref(sha2)'`);
```

Comment:
Make the instruction more clear. Like
Input a ref(sha1) to compare with ${currentRef} or 'ref(sha1)..ref(sha2)' to compare with two commits